forms/review.form.php

```diff
@@ -44,46 +44,7 @@
 		// if userEmail is empty, add error message to array to be displayed later
 		$error['userEmail'] = 'Please Enter you Email';
 	}
-
```

Comment:
Why are we deleting any of this?!

Reply:
This escapes me. Not certain why anything was deleted. Especially off of this file???

Reply:
We should find out when and why. Removing any of this is going to leave us dead in the water if the user turns off js